PizzaPyWebApp/templates/partials/landing/about_us.html

```diff
@@ -0,0 +1,75 @@
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">Web Development</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg class="w-5 h-5 text-yellow-300" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">Cloud Computing</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">Artificial Intelligence</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">Internet of Things</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">Game Development</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">Blockchain</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">System Design</span>
+                    </li>
+                    <li class="flex items-center">
+                        <svg fill="none" class="w-5 h-5 text-yellow-500" viewBox="0 0 24 24">
+                            <use xlink:href="../../../static/css/icons.svg#checks"></use>
+                        </svg>
+                        <span class="ml-2">and many more!</span>
+                    </li>
+                </ul>
+            </div>
+            <button class="px-4 py-2 font-bold bg-purple-600 rounded hover:bg-purple-800">
```

Comment:
Should be an `<a href="/about_us">` tag.

Also, add proper spacing to match the Figma design as shown here:
![image](https://github.com/pizzapy/pizzapy-website/assets/105530193/af4fcb7f-902a-43cf-b02c-b6ccf26572ff)

**Quick Figma Tip**: Click on an element and hold <kbd>ALT</kbd> while hovering on other elements to see the spacing between them.